data-prepper-test-common/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'java'
+}
+
+group 'org.opensearch.dataprepper'
+version '1.3.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.hamcrest:java-hamcrest:2.0.0.0'
```

Comment:
This dependency appears to be a few years old. Hamcrest is now on 2.2.

Should you use this dependency? https://search.maven.org/artifact/org.hamcrest/hamcrest/2.2/jar